play_tester.py

```diff
@@ -14,11 +14,12 @@
 pp.pprint(FANTASY_1)
 pp.pprint(FANTASY_2)
 
-# print("\n-----Wave 02 user_data-----")
-# pp.pprint(clean_wave_2_data())
+create_movie("title", "genre", "rating")
```

Comment:
Just FYI, in the future, if you make changes to a sandbox file like this, you don't need to add it to the staging area/commit it and push up the changes.

It's no problem that you did so here since this isn't a PR that will get merged into the project, but if your changes were to be merged into a project then someone reviewing your code would probably ask you to remove these changes from the PR so they don't get merged into the main code base.